pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -37,6 +38,10 @@ import (
 	netutils "k8s.io/utils/net"
 )
 
+const (
+	os_windows = string(core.Windows)
```

Comment:
Isn't core.Windows a string already?

Reply:
`core.Windows` is type of OSName, which is a Type definition of `string`